src/components/Favorites.jsx

```diff
@@ -1,5 +1,19 @@
-import React from 'react';
+import styled from 'styled-components';
+import Product from './Product';
 
 export default function Favorites(props) {
-  return <footer>Favorites</footer>;
+  const {favoriteItems, onBookmark} = props;
+  return (
+    <StyledArticle>
+      <h3>Favorites</h3>
+      <div>{favoriteItems.length === 0 && <div> Favorites empty</div>}</div>
+      {favoriteItems &&
+        favoriteItems.map(product => <Product key={product.id} product={product} onBookmark={onBookmark}></Product>)}
```

Comment:
I think you don't need a closing `</Product>`. 
`<Product key={product.id} product={product} onBookmark={onBookmark}/>` should work as well.